dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -96,7 +88,7 @@
     QueuedChannel(
             LimitedChannel delegate,
             int maxQueueSize,
-            TaggedMetricRegistry metrics) {
+            TaggedMetricRegistry registry) {
```

Comment:
Can this take a `DispatcherMetrics` instead

Reply:
👍